# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/awake/hotfix-setup.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- awake-1.0/setup.py.vanilla	2026-10-06 18:08:51.576040921 +0300
+++ awake-1.0/setup.py	2026-10-06 18:10:34.049817020 +0300
@@ -1,5 +1,5 @@
 import os
-from distutils.core import setup  
+from setuptools import setup  
 
 
 REPODIR = os.path.dirname(os.path.realpath(__file__))
